src/orion/algo/hyperband.py

```diff
@@ -293,19 +346,20 @@ def promote(self, num):
 
         return samples
 
-    def sample(self, num):
-        samples = []
+    def sample(self, num: int) -> list[Trial]:
+        samples: list[Trial] = []
+        assert self.brackets is not None
```

Comment:
Another

Reply:
Fixed in 35ce6867769d77028c87f7bb015adfd349315f85